docs/UserGuide.md

```diff
@@ -164,6 +150,20 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in JobFestGo.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding a tag : `add_tag`
+
+Adds a tag to JobFestGo.
+
+Format: `add_tag t/TAG_NAME`
+
+* Adds the tag with the specified tag name `TAG_NAME`.
+* The tag name **must no already exist** in the JobFestGo.
```

Comment:
I think you forgot to remove the 'no' and perhaps remove the additional 'the'

Reply:
Good catch! I have updated accordingly.